packages/website/src/components/CurriculumSection/TechIcons.tsx

```diff
@@ -0,0 +1,55 @@
+import {
```

Comment:
💭 To follow our previous conventions you can name this file `Curriculum.data.ts`. Also, you can separate the `techNames` array into two pieces - one for the top carousel and one for the bottom one. This way we will be sure that the user never sees identical technologies on both bars.